src/js/components/AppListComponent.jsx

```diff
@@ -123,6 +126,10 @@ var AppListComponent = React.createClass({
       "hidden": state.fetchState !== States.STATE_ERROR
     });
 
+    var unauthorizedClassSet = classNames({
```

Comment:
I'd prefer it if we reused the other `<tr>` with `noAppsClassSet` and just change the error message. What do you think?